Views/Sensors/Edit.cshtml

```diff
@@ -40,4 +40,8 @@
             </div>
         </div>
     </div>
-</div>
\ No newline at end of file
+    <div class="col-12 col-sm-12 col-md-5 col-lg-3">
+       
+        
+    </div>

```

Comment:
Is this empty column necessary?

Reply:
For now yes, I'm planning to add information later, I'll add comment on that.